test/image/mocks/gl2d_parcoords_1.json

```diff
@@ -0,0 +1,29 @@
+{
+  "layout": {
+    "width": 284,
+    "height": 400
+  },
+
+  "data": [{
+
+    "type": "parcoords",
```

Comment:
Thanks! Commits just pushed to fix it. I'll change one of the image test cases to a non-default `domain` to avoid future regression.
![image](https://cloud.githubusercontent.com/assets/1548516/22749908/30b41a14-ee2f-11e6-872d-671903ecce04.png)